scripts/semconv/templates/registry/common.j2

```diff
@@ -0,0 +1,46 @@
+{%- macro file_header() -%}
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{% endmacro -%}
+
+{%- macro to_const_name(attr_name) -%}
+{{attr_name | upper | replace('.', '_')}}
+{%- endmacro %}
+
+{%- macro to_class_name(attr_name) -%}
+{{attr_name | upper | title | replace('_', '') | replace('.', '')}}
+{%- endmacro %}
+
+{%- macro str_or_empty(str) -%}
+{% if str is defined and str is not none %}{{str}}{% else %}{{""}}{% endif %}
+{%- endmacro %}
+
+{%- macro remove_trailing_dots(str) -%}
+{%- if str[-1:] == '.' -%}{{ remove_trailing_dots(str[:-1]) }}{%- else -%}{{ str }}{%- endif -%}
+{%- endmacro -%}
+
+{%- macro to_docstring(str, prefix) -%}
+{{remove_trailing_dots(str | trim(' \n')) | comment_with_prefix(prefix)}}
+{%- endmacro %}
+
+{%- macro import_deprecated(semconv) -%}
+    {%- if (semconv | selectattr("deprecated", "defined") | rejectattr("deprecated", "none") | list | count > 0) or (ctx.filter == "any" and semconv | selectattr("stability", "equalto", "stable") | list | count > 0) -%}
```

Comment:
The test can probably be replaced by the custom Jinja test `deprecated`.

```
{% if semconv is deprecated ... %}...
```

`stable`, `experimental` are also supported.

Reply:
Here i'm checking if any of the attributes passed in `semconv` (it's a list) are deprecated. 
I.e. I need a filter, not a test. I assume we can add all existing tests as filters as well?

Reply:
ah, `semconv | select("deprecated")` would work